### DISCUSSION TITLE
Issue when attempting to compile source code

### DISCUSSION THREAD

**Discussion by Just-Chekin** [AndreyPavlenko/Fermata#98] [General]
Hello,

When I attempt to compile the code for the program, I get the following error from line 33 of build.gradle...

A problem occurred evaluating project ':control'.
> Could not get unknown property 'release' for SigningConfig container of type org.gradle.api.internal.FactoryNamedDomainObjectContainer.

I was curious if there is a known fix for this issue.  Thanks.

**Comment by AndreyPavlenko** [comment]
Works for me.

git clone --recurse-submodules https://github.com/AndreyPavlenko/Fermata.git
cd Fermata
export ANDROID_SDK_ROOT=~/android/android-sdk/
./gradlew -p control assembleRelease
./gradlew packageAutoReleaseUniversalApk -PABI=arm64-v8a
./gradlew fermata:bundleAutoRelease -PABI=arm64-v8a # Use this if you need aab
